### PR TITLE
Fix `app` option on Windows

### PR DESCRIPTION
**Pull request by Hongarc**
Reopen #162

I moved check `isWsl` in `if (!isWsl)` 

https://github.com/Hongarc/open/commit/3530dcd061498b597819fd21ed00282b082989dd

Should I keep check it in `if(options.app)`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -77,6 +77,7 @@ module.exports = async (target, options) => {
 			// As a result, all double-quotes are stripped from the `target` and do not get to your desired destination.
 			target = `"${target}"`;
 			childProcessOptions.windowsVerbatimArguments = true;
+			options.app = options.app && `"${options.app}"`;
```
Comment (sindresorhus) [inline index.js]: A `if` statement would be more readable.

File: index.js [hunk 2]
```diff
@@ -88,8 +89,6 @@ module.exports = async (target, options) => {
 				const windowsPath = await wslToWindowsPath(options.app);
 				options.app = windowsPath;
 			}
-
-			cliArguments.push(options.app);
```
Comment (sindresorhus) [inline index.js]: Why are you removing this?

**Comment by sindresorhus** [comment]
Have you manually tested this change?

File: index.js (hunk 2 above)
Comment (Hongarc) [reply index.js]: I moved it `if`, and mistake to remove it

**Comment by Hongarc** [comment]
> Have you manually tested this change?

I used this code:

```js
open('./test.js', {app: 'C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe'})
```

success to open `test.js` file it in chrome

**Comment by Hongarc** [comment]
Fixed #158, it should be closed

**Pull request merged**